weave/trace/op.py

```diff
@@ -85,10 +85,10 @@ async def anext(obj: AsyncIterator[V], default: V | None = None) -> V:
 logger = logging.getLogger(__name__)
 
 
-CALL_CREATE_MSG = "Error creating call:\n{}"
-ASYNC_CALL_CREATE_MSG = "Error creating async call:\n{}"
-ON_OUTPUT_MSG = "Error capturing call output:\n{}"
-UNINITIALIZED_MSG = "Warning: Traces will not be logged. Call weave.init to log your traces to a project.\n"
+CALL_CREATE_MSG = "Failed to log call to Weave server (your code will continue to run). Error details:\n{}"
```

Comment:
Do we really need to mention `your code will continue to run`? Firstly, that is the expected behavior, and secondly, it should be pretty obvious to the user since the program has proceeded.